Dockerfile

```diff
@@ -1,23 +1,32 @@
 FROM registry.opensource.zalan.do/stups/python:3.5-cd26
 MAINTAINER lothar.schulz@zalando.de
 
-USER root
```

Comment:
As stated in [dockerfile_best_practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#user) using su or sudo is not recommended, instead, after adding the user you could just use "USER application"